.gitignore

```diff
@@ -1,4 +1,6 @@
 node_modules/
+!docs/app/showcase/charts/dist/
```

Comment:
Some bower component files dont seem to be checked in:

![image](https://cloud.githubusercontent.com/assets/20795331/25003127/d3d5056c-2045-11e7-8d41-27e65f39d931.png)

and on charts page:

![image](https://cloud.githubusercontent.com/assets/20795331/25003161/fc4ee756-2045-11e7-820e-54e3574af6ca.png)